lesson-4/script.js

```diff
@@ -0,0 +1,137 @@
+  // Вивести на сторінку в один рядок через кому числа від 10 до 20.
+
+  let result = "";
+
+  for (let i = 10; i <= 20; i++) {
+    if (i === 20) {
+      result = `${result + i}`;
+    } else {
+      result = `${result + i}, `
+    }
+  }
+  console.log(result);
```

Comment:
I recommend you to work with array in this case. And console it using Array.join method(), with separator ', '
```suggestion
  let result = [];

  for (let i = 10; i <= 20; i++) {
    result.push(i);
  }
  console.log(result.join(', ')); // '10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20'
```